go/teams/list.go

```diff
@@ -595,31 +600,16 @@ func AnnotateInvitesNoPUKless(mctx libkb.MetaContext, team *Team,
 			}
 
 			if pkg.NormalizedUsername.IsNil() {
-				return nil, fmt.Errorf("failed to get username from UIDMapper for uv %v", uv)
+				return nil, fmt.Errorf("failed to get username from UIDMapper for uv %v", inviteeUV)
 			}
 
-			details := keybase1.TeamMemberDetails{
-				Uv:       uv,
-				Username: pkg.NormalizedUsername.String(),
-				NeedsPUK: true,
-				FullName: fullName,
-				Status:   status,
-			}
-
-			if members != nil {
-				switch invite.Role {
-				case keybase1.TeamRole_OWNER:
-					members.Owners = addKeybaseInviteToRes(mctx.Ctx(), details, members.Owners)
-				case keybase1.TeamRole_ADMIN:
-					members.Admins = addKeybaseInviteToRes(mctx.Ctx(), details, members.Admins)
-				case keybase1.TeamRole_WRITER:
-					members.Writers = addKeybaseInviteToRes(mctx.Ctx(), details, members.Writers)
-				case keybase1.TeamRole_READER:
-					members.Readers = addKeybaseInviteToRes(mctx.Ctx(), details, members.Readers)
-				}
-			}
-			// Continue to skip adding this invite to annotatedInvites
-			continue
+			displayName = keybase1.TeamInviteDisplayName(pkg.NormalizedUsername.String())
```

Comment:
we're adding to invites map here; i.e., this function is not "Lying" about kb invites